examples/scripts/reward_modeling.py

```diff
@@ -12,15 +12,29 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""
```

Comment:
Now need to add a launch command because we need to fill in things like `output_dir`